docs/imviz/plugins.rst

```diff
@@ -223,6 +223,10 @@ catalog dropdown menu.
     This plugin is still under active development. As a result, the search only uses the SDSS DR17 catalog
     and works best when you only have a single image loaded in a viewer.
 
+To load a catalog from an ecsv file, choose "From File..." and choose a valid file.  The file must be able
+to be parsed by `astropy.table.Table.read` and contain a column labeled 'sky_centroid'.  Clicking
```

Comment:
I am still not convinced that "sky_centroid" is a universal convention for column name containing SkyCoord objects. Especially your test seems to be parsing a cached SDSS query, so that column must have been added internally by the plugin itself. Just where is this JWST ECSV file format defined?

Reply:
the example data file I grabbed from MAST (`jw02731-o001_t017_nircam_f444w-f470n_cat.ecsv`) to write this did... but if you find any that don't, send them my way and I'll try to make the logic more flexible to handle them as well.

The only thing I found in docs online about the format of the files does not describe individual columns: https://jwst-docs.stsci.edu/understanding-jwst-data-files/jwst-data-formats#JWSTDataFormats-ecsvECSVformat

Reply:
Here is where the format is described: https://jwst-pipeline.readthedocs.io/en/latest/jwst/source_catalog/main.html#output-products

Reply:
Great, thanks! We should link to that doc, if not already.

Reply:
is it a permanent link that won't change?  If so - I can add it to the plugin docs.  But otherwise, I think the feedback in the UI should be sufficient for now and we can generalize as needed to other file formats down the road.

Reply:
This is `jwst` doc, so we could use intersphinx. RTD would fail if they move the `:ref:` and I can go bug Nadia or Howard. No link is permanent. But a link is better than no link.

Reply:
Ok, I added an external link for now (don't see any refs in their source rst file for that section).